packages/adapter/addon/rest.js

```diff
@@ -1414,4 +1419,23 @@ function ajaxOptions(options, adapter) {
   return options;
 }
 
+if (DEPRECATE_NAJAX) {
+  RESTAdapter.reopen({
+    /**
+      @method _najaxRequest
+      @private
+      @param {Object} options jQuery ajax options to be used for the najax request
+    */
+    _najaxRequest(options) {
+      if (hasNajax) {
+        najax(options);
+      } else {
+        throw new Error(
+          'najax does not seem to be defined in your app. Did you override it via `addOrOverrideSandboxGlobals` in the fastboot server?'
+        );
+      }
+    },
+  });
+}
+
```

Comment:
Ideally this would be done in native class syntax, but I can certainly follow up after this with another PR to migrate